### PR TITLE
1716 banked months updater

### PR DESCRIPTION
**Pull request by IlseFerris**
Updated ACTIONS - BANKED MONTHS UPDATER for new directions. Team chat will have the gist of the new changes. Training case #328740 can be used to test.

### CODE REVIEW

File: actions/banked-months-updater.vbs [hunk 1]
```diff
@@ -270,34 +273,36 @@ For i = 0 to Ubound(banked_months_array, 2)
...
 		banked_months_array(abawd_month_eval_const, i) = "All ABAWD/TLR months used."
 	Elseif abawd_counted_months < 3 then 
 		banked_months_array(used_all_abawd_mo_const, i) = False
-		banked_months_array(abawd_month_eval_const, i) = "Only " & abawd_counted_months & " have been used. This member does not require banked months yet."
-	Else 
-		banked_months_array(member_in_error_const, i) = banked_months_array(member_number_const, i) & " " & banked_months_array(member_first_name_const, i) & " has used " & abawd_months_count & " abawd months. Only 3 are allowed. Updates are needed to this STAT/WREG and/or ABAWD Tracking Record before the script can support this case." 
-		banked_months_array(abawd_month_eval_const, i) = abawd_counted_months & " have been used. Only 3 are allowed. Updates are needed."
+		banked_months_array(abawd_month_eval_const, i) = "Only " & abawd_counted_months & " have been used."
+        banked_months_array(member_in_error_const, i) = banked_months_array(member_number_const, i) & " " & banked_months_array(member_first_name_const, i) & " has used " & abawd_months_count & " abawd months. This member does not require banked months yet."
+	'Else
```
Comment (MeganGeissler) [inline actions/banked-months-updater.vbs:283]: Thinking abawd_months_count should be abawd_counted_months

File: actions/banked-months-updater.vbs [hunk 2]
```diff
@@ -308,15 +313,18 @@ Show_dialog = False
...
+        spec_memo = True 
+        case_note = True 
 	End if 
 
-	If banked_months_array(used_all_banked_mo_const, i) = False then spec_memo = True 
-	If update_wreg = False then spec_memo = False 
-	If update_wreg = True then case_note = True 
+	'If banked_months_array(used_all_banked_mo_const, i) = False then spec_memo = True 
+	'If update_wreg = False then spec_memo = False 
+	'If update_wreg = True then case_note = True
```
Comment (MeganGeissler) [inline actions/banked-months-updater.vbs:327]: Line 356 Could use a little extra room if we anticipate real cases having this many months listed as it is cutting off the last month
banked months

**Review by MDR22 — APPROVED** [review]
Same fixes as noted by Megan, otherwise looks good!

File: actions/banked-months-updater.vbs (hunk 1 above)
Comment (IlseFerris) [reply actions/banked-months-updater.vbs:283]: Good catch!

File: actions/banked-months-updater.vbs (hunk 2 above)
Comment (MeganGeissler) [reply actions/banked-months-updater.vbs:327]: banked months
Also wondering for line 358 if banked_months_array(banked_mo_string_const, i)) should default to 0 or something when the string is blank

**Review by MeganGeissler — APPROVED** [review]
Nothing mission critical but added a few comments, all works well!

**Pull request merged**